### DISCUSSION TITLE
Does adjusting the sidetone on Corsair VOID RGB ELITE make settings persist in the headset?

### DISCUSSION THREAD

**Discussion by g0hww** [Sapd/HeadsetControl#225] [Q&A] [answered]
Just bought one of these as the headband snapped on my old Turtle Beach headset.
I will use it primarily for WFH on a locked down Win10 machine, so no chance of running the ICUE software on that.
A secondary use will be using it on my Ubuntu 20.04 box.
I seem to have no sidetone audio after cycling sidetone on/off/on, etc. 

Questions ...
1) Is it really the case that sidetone level is 0 by default?
2) If I somehow manage to increase the sidetone volume on Linux with this tool, will it persist when I use the headset subsequently on Windows?

**Comment by Sapd** [comment, marked as answer]
> If I somehow manage to increase the sidetone volume on Linux with this tool, will it persist when I use the headset subsequently on Windows?

At least for my Corsair headset it does never save the settings. They all need to be transferred to the headset when its turned on.

You can also use HeadsetControl on Windows if thats an option.

**Comment by g0hww** [comment]
Thanks for response. 
Regarding Headset Control, it can only be an option if it is installed by default on Win10. The only way i get to run code on my work puter that isn't authorised (and that's not a road to take) is if I write it myself.  I only use Windows at work and long since gave up caring about how it works or what it ships with by default.
I'll give this code a whizz as I will want sidetone on Linux, but I will spend much more time using the headset for work than for home purposes, so I am once again disappointed by Corsair hardware/software combinations (having eaten my way through more than my fair share of Corsair AIO coolers before switching back to fans). 
To be totally fair though, during the 5 minutes I spent hurriedly choosing a new headset on Amazon, i never saw any mention of the sidetone feature, so I did expect to be disappointed before I even ordered them.
Cheers.
